### PR TITLE
feat: cc-0033 change price bar to use chakra box

### PR DESCRIPTION
**Pull request by qiying417**
Rewrite the price bar to use Chakra Box component instead of Drawer.
Create transition effect for open/close price bar use Chakra Collapse.
image
image

Responsive Design for iPad included.
image

### CODE REVIEW

**Review by Jzeva — APPROVED** [review]
LGTM

File: src/components/PriceBar/index.tsx [hunk 1]
```diff
@@ -8,50 +8,37 @@ import TileColorBoard from "./TileColorBoard";
...
-          backgroundColor="#fff"
-          sx={{
-            width: "calc(100% - 98px)",
-            marginLeft: "98px",
-            marginBottom: "42px",
-          }}
-        >
-          {drawerButton(drawerController)}
-          <DrawerBody boxShadow="0px -5px 10px -5px lightgrey" p="0">
+    <Box position="fixed" bottom="42px" right="0px " width="calc(100% - 98px)" zIndex="1600">
```
Comment (tatoMa) [inline src/components/PriceBar/index.tsx:29]: Please remove the empty space in the right props.

**Review by tatoMa — APPROVED** [review]
LGTM

File: src/styles/components/drawerButtonStyles.ts [hunk 2]
```diff
@@ -1,26 +1,13 @@
...
-  },
-};
-
 interface DrawerButtonProps {
   isOpen: boolean;
 }
 
-export const DrawerButton = {
+const drawerButtonStyles = {
   baseStyle: ({ isOpen }: DrawerButtonProps) => ({
```
Comment (Fuchih) [inline src/styles/components/drawerButtonStyles.ts:6]: You can remove the interface and isOpen as these codes don't need anymore.

File: src/components/PriceBar/index.tsx [hunk 3]
```diff
@@ -8,50 +8,37 @@ import TileColorBoard from "./TileColorBoard";
 
 const PriceBar: React.FC = () => {
   const [isOpen, setIsOpen] = React.useState<boolean>(true);
-  const drawerBtnStyles = useStyleConfig("DrawerButton", { isOpen });
-
-  const drawerController = () => setIsOpen(!isOpen);
+  const drawerBtnStyles = useStyleConfig("drawerButtonStyles", { isOpen });
```
Comment (Fuchih) [inline src/components/PriceBar/index.tsx:11]: You can remove { isOpen }

File: src/components/PriceBar/index.tsx (hunk 1 above)
Comment (qiying417) [reply src/components/PriceBar/index.tsx]: Thanks Tao zong, fixed :)

File: src/styles/components/drawerButtonStyles.ts (hunk 2 above)
Comment (qiying417) [reply src/styles/components/drawerButtonStyles.ts]: Thanks Jin, fixed :)

File: src/components/PriceBar/index.tsx (hunk 3 above)
Comment (qiying417) [reply src/components/PriceBar/index.tsx]: Thanks Jin, fixed 👍

**Pull request merged**